provider/lxd/environ.go

```diff
@@ -30,7 +32,10 @@ import (
 
 var _ environs.HardwareCharacteristicsDetector = (*environ)(nil)
 
-const bootstrapMessage = `To configure your system to better support LXD containers, please see: https://documentation.ubuntu.com/lxd/en/latest/explanation/performance_tuning/`
+const (
+	bootstrapMessage = `To configure your system to better support LXD containers, please see: https://documentation.ubuntu.com/lxd/en/latest/explanation/performance_tuning/`
+	profileNotFound  = "Profile not found"
```

Comment:
Let's comment this to explain we need it because lxd api doesn't give us typed errors